src/Qwiq.Core/IWorkItem.cs

```diff
@@ -212,5 +212,7 @@ public interface IWorkItem
         /// An ArrayList of the fields in this work item that are not valid.
         /// </returns>
         IEnumerable<IField> Validate();
+
+        void ApplyRules(bool copyChangedByFromLatest = false);
```

Comment:
I'm not actually sure what `copyChangedByFromLatest` does? Is there a better variable name here?